contracts/NFTs/VehicleId.sol

```diff
@@ -1,9 +1,12 @@
 //SPDX-License-Identifier: BUSL-1.1
 pragma solidity ^0.8.13;
 
-import "./Base/MultiPrivilege/MultiPrivilege.sol";
+import "../interfaces/IDimoRegistry.sol";
+import "./Base/MultiPrivilege/MultiPrivilegeTransferable.sol";
 
 contract VehicleId is Initializable, MultiPrivilege {
+    IDimoRegistry private _dimoRegistry;
```

Comment:
Are we just adding this in case we need it later?

Reply:
That is the idea, bc the other NFTs also have it ("uniformity"). Maybe it is dumb

Reply:
Seems fine to me.